### DISCUSSION TITLE
how to add `docsVersionDropdown` to specific route only

### DISCUSSION THREAD

**Discussion by iamayushdas** [facebook/docusaurus#4810] [Q&A]
how to add `docsVersionDropdown` to specific route only. i want that docsVersionDropdown to be visible only on a particular page.
Help me to achieving that

**Comment by slorber** [comment]
It is currently not possible but you can track this issue: https://github.com/facebook/docusaurus/issues/4389

**Reply by iamayushdas** [reply]
If i want to show the dropdown only to the particular route and hide from all other routes, how can i achieve it?

**Reply by slorber** [reply]
we don't provide any simple way to do that, cf the feature request above

The only option would be to swizzle the docsVersionDropdown component and implement your own

**Comment by iamayushdas** [comment]
If i want to show the dropdown only to the particular route and hide from all other routes, how can i achieve it?
